localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
+ 
+ innovation_dragonhawk:0 "Dragonhawk Taming"
+ innovation_dragonhawk_desc:0 "The dragonhawks are deft predators that rend apart their foes with razor-sharp talons. They are among the deadliest known predators, able to spot prey from great distances, rush in, and rend with both talon and beak. In addition, they're capable of breathing fire. Our tamers have successfully learned how to communicate and tame these flying creatures."
+
+ innovation_farstriders:0 "Farstriders"
+ innovation_farstriders_desc:0 "The Farstriders are an elite group of elven rangers active in Quel'Thalas. Intimately bound to the wildlands, and with a deep respect for the law, the rangers that compose the Farstriders' ranks remain noble, traditional, and valiant protectors of elven society."
+    
+ innovation_fel_guard:0 "Felguards"
+ innovation_fel_guard_desc:0 "Felguard are fearless and unwaveringly loyal to their master's godlike will. They are almost always seen with heavy, two-handed weaponry. They are slow to strike, but hit quite hard, and are more agile than they look."
+    
+ innovation_footmen:0 "Footmen"
+ innovation_footmen_desc:0 "Advancing with raised shields in close formation, the footmen are proven soldiers of the Eastern Kingdoms. Guarded by their large shields and heavy armor and excellent at hand-to-hand combat, they can easily endure volleys of arrows and break enemy charges."
+ 
+ innovation_forsaken_ranged:0 "Forsaken Ranged Tactics"
+ innovation_forsaken_ranged_desc:0 "With our unique ranged armaments, we have produced regiments like the Dark Rangers."
+    
+ innovation_gladiator:0 "Gladiators"
+ innovation_gladiator_desc:0 "Forced to fight for survival, gladiators are professional warriors, capable of great feats in combat."
+
+ innovation_golem:0 "Golem Construct"
+ innovation_golem_desc:0 "Golems are constructs, magically wrought automatons of bones, flesh, metal, or stone and are generally thought to be mindless. They are built generally as guardians or protectors of a person, place, or thing."
+ 
+ innovation_gryphon:0 "Gryphon Taming"
+ innovation_gryphon_desc:0 "Gryphons are majestic creatures of the skies, and are, much like wyverns, more than just beasts. They both are intelligent as well as cunning. Our tamers have successfully learned how to communicate with and tame these flying creatures."
+ 
+ innovation_night_elf_aerial:0 "Darnassian Aerial Tactics"
+ innovation_night_elf_aerial_desc:0 "With our unique aerial armaments, we have produced regiments like the Faerie Dragons and Chimaeras."
+ 
+ innovation_night_elf_ground:0 "Darnassian Ground Tactics"
+ innovation_night_elf_ground_desc:0 "With our unique ground armaments, we have produced regiments like the Watchers and Mountain Giants."
+ 
+ innovation_night_elf_mounted:0 "Darnassian Mounted Tactics"
+ innovation_night_elf_mounted_desc:0 "With our unique mounted armaments, we have produced regiments like the Huntress."
```

Added plural, changed unique mounted armaments to our unique mounted armaments, as I don't think you can call an animal an armament. 
```suggestion
 innovation_night_elf_mounted_desc:0 "With our unique companions, the nightsabers, we have produced regiments like the Huntresses."
```